src/components/DecodedCalldataTree.tsx

```diff
@@ -57,28 +57,58 @@ function CalldataTreeNode({
     return (
       <span className={className}>
         <code>
-          {node.name ? (
-            <b className="text-deth-pink"> {node.name} </b>
+          {node.type.match(/[[*\]]/) ? (
```

Comment:
Can we please have a convention that we name all the regexes that are not trivial?
I think this matches: `[`, `]`, `*` but it takes some time to mentally parse and I still don't get why there is a star in a char set.
Let's have smth like: `arrayOrStructRegex` or whatever this actually is.

Reply:
I have a stronger ask.
Could we comment all non trivial regexes with examples? :D

Reply:
fine I will extract it from here 😆